test/Microsoft.Health.Fhir.Tests.E2E/Rest/BasicAuthTests.cs

```diff
@@ -123,22 +167,13 @@ public async Task WhenGettingAResource_GivenAUserWithReadPermissions_TheServerSh
 
         [Fact]
         [Trait(Traits.Priority, Priority.One)]
-        public async Task WhenUpdatingAResource_GivenAUserWithUpdatePermissions_TheServerShouldReturnSuccess()
+        public async Task WhenGettingAResource_GivenAClientWithWrongAudienceAuthToken_TheServerShouldReturnUnAuthorized()
```

Comment:
A few things on this test:
- This won't run the same when the tests are targeting AAD as the identity provider. You'll probably get a 401 instead of a 403. 
- If for some reason the Client has already gotten a token for the NativeClient, it won't attempt to get one with the wrong audience.
- Does updating the Audience like this change it for all subsequent calls?